pkg/sync/v6/data/ImageStream.image.openshift.io/openshift/dotnet-runtime.yaml

```diff
@@ -9,21 +9,6 @@ spec:
   lookupPolicy:
     local: false
   tags:
-  - annotations:
-      description: 'RETIRED: Run .NET Core applications on RHEL 7. For more information
-        about using this image, including OpenShift considerations, see https://github.com/redhat-developer/s2i-dotnetcore/tree/master/2.0/runtime/README.md.'
-      iconClass: icon-dotnet
-      openshift.io/display-name: .NET Core 2.0 Runtime
-      supports: dotnet-runtime
-      tags: hidden,runtime,.net-runtime,dotnet-runtime,dotnetcore-runtime
-      version: "2.0"
-    from:
-      kind: DockerImage
-      name: registry.redhat.io/dotnet/dotnet-20-runtime-rhel7:2.0
-    importPolicy: {}
-    name: "2.0"
-    referencePolicy:
-      type: Local
```

Comment:
Are these removals just an unrelated content update?

Reply:
yes. script ran and produced it